### DISCUSSION TITLE
Enable right click / secondary click to open menu items

### DISCUSSION THREAD

**Discussion by kousikmitra** [Kong/insomnia#5881] [Ideas] [answered]
Right now to open menu on sidebar items one has to click on the dropdown icon. since the right click doesn't do any operation  on sidebar items It would be very convenient  to enable secondary/right click button to open menu.

**Comment by nettnikl** [comment]
This is a quite common pattern, usability-wise pretty much a must-have, i think it should be possible even with Electron, but maybe someone with experience can describe whether/how hard it is to do?

**Comment by subnetmarco** [comment]
I am not understand this request, can you be more descriptive?

**Reply by sky96111** [reply]
Hello, I described it in discussion  https://github.com/Kong/insomnia/discussions/7290

**Comment by subnetmarco** [comment]
We are looking into this, I agree it would be nice to have (cc @gatzjames )

**Comment by jackkav** [comment]
Available in latest beta

**Reply by subnetmarco** [reply, marked as answer]
Like @jackkav mentioned this will ship in 9.3.3 soon (currently available in the beta).
